doc/index.rst

```diff
@@ -1,38 +1,45 @@
-Autocrypt support for mail agents
-=================================
+muacrypt: help mail agents manage automated encryption
+======================================================
 
 .. note::
 
     There is a `tentative name change consideration
     <https://github.com/hpk42/py-autocrypt/issues/34>`_ for this project
-    which would result in change of links.
+    which is to result in change of links, names.
 
 py-autocrypt provides a command line tool and a Python API to help
```

Comment:
muacrypt instead of py-autocrypt here? Not sure which heuristic you are using for which usages to rename and which to keep.

Reply:
this and many other places need re-naming.  i didn't follow through because it would mean doing it all again if the name changes.